lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -281,6 +285,11 @@ module InternalConstants
     # Cloud Logging API.
     config_param :use_grpc, :bool, :default => false
 
+    # Whether valid entries should be written even if some other entries fail
+    # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
+    # the Cloud Logging API. This is highly recommended.
+    config_param :partial_success, :bool, :default => false
```

Comment:
This currently only works for the non-gRPC path. Should we add some sort of validation that prints a warning that this key will be ignored if `use_grpc` is on?

Reply:
Added.

Reply:
Let's also add a note in the comment that this only works without `use_grpc`. We can remove that note whenever we remove the check.

Reply:
Added.